tests/ReaderTest.cc

```diff
@@ -606,3 +653,5 @@ TEST(ReaderTest, testReceiveAfterSeek) {
 
     client.close();
 }
+
+INSTANTIATE_TEST_CASE_P(Pulsar, ReaderTest, ::testing::Values(true, false));
```

Comment:
```suggestion
INSTANTIATE_TEST_SUITE_P(Pulsar, ReaderTest, ::testing::Values(true, false));
```

`INSTANTIATE_TEST_CASE_P` is deprecated.